base/applications/taskmgr/perfdata.c

```diff
@@ -44,7 +44,7 @@ LARGE_INTEGER                              liOldSystemTime = {{0,0}};
 SYSTEM_PERFORMANCE_INFORMATION             SystemPerfInfo;
 SYSTEM_BASIC_INFORMATION                   SystemBasicInfo;
 SYSTEM_FILECACHE_INFORMATION               SystemCacheInfo;
-SYSTEM_HANDLE_INFORMATION                  SystemHandleInfo;
+ULONG                                      SystemNumberOfHandles;
```

Comment:
This needs to be initialized, otherwise it will be garbage if `NtQuerySystemInformation` fails

Reply:
Globals default to `0`, don't they?

Reply:
Static's do, others don't.

Reply:
(File) Globals, whether `static` or not, behave same as `static` (function) locals, don't they?

Reply:
It does indeed seem that file globals still have static storage without the static keyword.